src/shared/cgroup-show.c

```diff
@@ -117,6 +119,44 @@ static int show_cgroup_one_by_path(
         return 0;
 }
 
+static int show_cgroup_name(
+                const char *path,
+                const char *prefix,
+                const char *glyph) {
+
+        _cleanup_free_ char *b = NULL;
+        bool delegate = false;
+        int r;
+
+        r = getxattr_malloc(path, "trusted.delegate", &b, false);
```

Comment:
Hmm, shouldn't we simply query pid1 then if running as unprivileged? Checking the xattr is nicer, but running as root just for that doesn't seems like too much.

Reply:
hmm, i wanted to avoid IPC for that since this way we can show the map for the whole cgroup tree, crossing all manager boundaries. That gets a lot more complex if we always have to figure out which manager to talk to (and also, we it's not even clear we *CAN* talk to the right manager, if it is inside some container down the tree)